lib/data/wrapper.js

```diff
@@ -158,14 +158,15 @@ const data = {
         client.healthcheck(log, (err, result) => {
             const respBody = {};
             if (err) {
-                log.error(`error from ${implName}`, { error: err });
-                respBody[implName] = {
+                log.error('error from sproxyd', { error: err });
```

Comment:
you could change 'mem' and 'file' in lines 16 and 19 above to 'dataMem' and 'dataFile'